src/main/resources/standardTestCases/globbingindex.test

```diff
@@ -0,0 +1,17 @@
+name: globbingindex
+testFormat: workflowsuccess
+
+files {
+  wdl: globbingindex/globbingindex.wdl
+}
+
+metadata {
+  workflowName: globbingindex
+  status: Succeeded
+  "outputs.globbingindex.globFileContent": "globFile"
+  "outputs.globbingindex.globArrayContent": "globArray"
+  "outputs.globbingindex.staticFileContent": "staticFile"
+  "outputs.globbingindex.staticFilesContent": "staticFile"
+  "outputs.globbingindex.staticRefContent": "staticFile"
+  "outputs.globbingindex.staticArrayContent": "staticArray"
```

Comment:
Depending on your OCD level, you might also want to add some `globLength = length(globArray)` outputs and verify that the globs are the correct length here.